src/core/public/index.ts

```diff
@@ -146,10 +149,14 @@ export {
   ChromeRecentlyAccessed,
   ChromeRecentlyAccessedHistoryItem,
   ChromeStart,
+  ContextContainer,
+  ContextProvider,
+  ContextSetup,
   DocLinksStart,
   ErrorToastOptions,
   FatalErrorInfo,
   FatalErrorsSetup,
+  Handler,
   HttpInterceptor,
```

Comment:
Can we make this handler name more specific to avoid clashes? Like `ContextHandler`?